Sources/Sentry/Public/PrivateSentrySDKOnly.h

```diff
@@ -41,6 +41,12 @@ typedef void (^SentryOnAppStartMeasurementAvailable)(
  */
 + (NSArray<SentryDebugMeta *> *)getDebugImages;
 
+/**
+ * Override SDK information.
+ */
++ (void)setSdkName:(NSString *)sdkName;
```

Comment:
`m`: I think it would be better to combine these into one method.

Reply:
Right now they are overriden setters for the two properties. I could add an extra method, but I don't think we can remove these 2 setters. And in that case I wonder if an extra method makes a lot of sense?

Reply:
Ugh I was looking at SentryMeta, not PrivateSentrySDKOnly 😰 So yea my comment didn't make sense. I think I need to eat something!

Done!